docs/README.rst

```diff
@@ -642,39 +665,42 @@ Supported Route Configurations
+|    |                                          |             | [#hm1]_   |                                                                                   |             |                                         |
++----+------------------------------------------+-------------+-----------+-----------------------------------------------------------------------------------+-------------+-----------------------------------------+
+|    | send                                     | string      | Required  | The send string to set in the health monitor. [#hm2]_                             | N/A         |                                         |
+|    |                                          |             | [#hm1]_   |                                                                                   |             |                                         |
++----+------------------------------------------+-------------+-----------+-----------------------------------------------------------------------------------+-------------+-----------------------------------------+
+|    | recv                                     | string      | Optional  | String or RegEx pattern to match in first 5,120 bytes of backend response.        | N/A         |                                         |
++----+------------------------------------------+-------------+-----------+-----------------------------------------------------------------------------------+-------------+-----------------------------------------+
+| virtual-server.f5.com/secure-serverssl        | boolean     | Optional  | Specify to validate the server-side SSL certificate of re-encrypt                 | false       | "true", "false"                         |
+|                                               |             |           | terminated routes.                                                                |             |                                         |
++-----------------------------------------------+-------------+-----------+-----------------------------------------------------------------------------------+-------------+-----------------------------------------+
+| virtual-server.f5.com/source-addr-translation | string      | Optional  | Source address translation type to apply to the virtual server.                   | automap     | automap, none, snat                     |
+|                                               |             |           | JSON object encoded string see virtualServer table entry for formatting examples. |             |                                         |
++-----------------------------------------------+-------------+-----------+-----------------------------------------------------------------------------------+-------------+-----------------------------------------+
 
 Please see the example configuration files for more details.
```

Comment:
Please provide examples of the correct usage in the example config file(s).

Reply:
Added examples.